gems/pending/VMwareWebService/MiqVimEventMonitor.rb

```diff
@@ -2,11 +2,12 @@
 require 'VMwareWebService/MiqVimInventory'
 
 class MiqVimEventMonitor < MiqVimInventory
-  def initialize(server, username, password, eventFilterSpec = nil, pgSize = 100)
+  def initialize(server, username, password, eventFilterSpec = nil, pgSize = 100, maxWait = 60)
```

Comment:
That is strange that it's in the defaults...I can't imagine how that could be reused by multiple event catchers.